app/views/main.scala.html

```diff
@@ -6,7 +6,12 @@
     Currently being used for digital pack page and to help transition to a consistent
     body width throughout the site. The extra classes will be removed when this is the case.
 *@
-@(title: String, jsVars: JsVars = JsVars.default, footer: Html = Html(""), bodyClasses: Seq[String] = Nil)(content: Html)
+@(
+    title: String,
+    jsVars: JsVars = JsVars.default,
+    isInternational: Boolean = false,
```

Comment:
Yep definitely :smile: . It's on my list of things to do, we do the same on Membership fwiw.